docker-admin-ui/scripts/upgrade.py

```
@@ -267,6 +267,33 @@ def update_scope(dn, old_name, new_name):
                     entry.attrs["jansScope"] = scopes
                 self.backend.modify_entry(entry.id, entry.attrs, **kwargs)
 
+        def add_scope(dn, new_scope):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `add_scope`

Add return type annotation: `None`

(ANN202)